backend/src/test/scala/com/softwaremill/adopttapir/starter/StarterServiceITTest.scala

```diff
@@ -49,27 +49,43 @@ object Setup {
       addDocumentation = docs,
       addMetrics = metrics,
       json,
-      scalaVersion
+      scalaVersion,
+      builder
     )
     starterDetails <- FormValidator.validate(starterRequest).toSeq
   } yield starterDetails
 
-  implicit class StarterDetailsWithDescribe(details: StarterDetails) {
-    lazy val describe: String =
-      s"${details.serverEffect}/${details.serverImplementation}/docs=${details.addDocumentation}/metrics=${details.addMetrics}/json=${details.jsonImplementation}/scalaVersion=${details.scalaVersion}"
-  }
-
   type TestFunction = Integer => IO[Unit]
 }
 
 object Timeouts {
+  val waitForScalaCliCompileAndUnitTest: FiniteDuration = 45.seconds
   val waitForPortTimeout: FiniteDuration = 90.seconds
   // wait for tests has to be longer than waiting for port otherwise it will break waiting for port with bogus errors
   val waitForTestsTimeout: FiniteDuration = waitForPortTimeout + 30.seconds
 }
 
+object Describers {
+  implicit class StarterDetailsWithDescribe(details: StarterDetails) {
+    lazy val describe: String =
+      s"${details.serverEffect}/${details.serverImplementation}/docs=${details.addDocumentation}/metrics=${details.addMetrics}" +
+        s"/json=${details.jsonImplementation}/scalaVersion=${details.scalaVersion}/builder=${details.builder}"
+  }
+
+  implicit class ThrowableWithDescribe(e: Throwable) {
+    def describe(): String = {
+      Using.Manager { _ =>
```

Comment:
According to api doc: 

> Performs an operation using a Manager, then closes the Manager, releasing its resources (in reverse order of acquisition).
Example:
 val lines = Using.Manager { use =>
   use(new BufferedReader(new FileReader("file.txt"))).lines()
 }
 
> If using resources which require an implicit Manager as a parameter, this method should be invoked with an implicit modifier before the function parameter:
Example:
 val lines = Using.Manager { implicit use =>
   new SafeFileReader("file.txt").lines()
 }
   
 Looks like resources are not cleared after execution of method. 
 
 I would expect something like: 
```
    def describe(): String = {
      Using.Manager { use =>
        val sw = use(new StringWriter())
        val pw = use(new PrintWriter(sw))
        e.printStackTrace(pw)
        sw.toString
      }.get
    }
```

Reply:
I have somehow misinterpreted it :/ thanks for catching it up!